### ISSUE TITLE
Is `safelist` working in the current version?

### ISSUE THREAD

**Issue by Mexican-Man** [Jax-p/vite-plugin-html-purgecss#1]
Everything else seems to be working fine. I'm using Bootstrap v5.1.1 ([here's the file](https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.css)). These rules are getting stripped out:
```css
.carousel-item-next:not(.carousel-item-start),
.active.carousel-item-end {
  transform: translateX(100%);
}

.carousel-item-prev:not(.carousel-item-end),
.active.carousel-item-start {
  transform: translateX(-100%);
}
```
so I've tried adding this to my vite config
```js
 htmlPurge({
  safelist: [ 'carousel-item-next', 'carousel-item-next', 'carousel-item-prev' ]
}),
```
as well as
```js
 htmlPurge({
  safelist: [ '/carousel/' ]
}),
```
with no luck. I'm new with purgecss, so I apologize if I'm missing something.

**Comment by Mexican-Man** [comment]
> @Mexican-Man were you able to successfully set this up?

Yes. I am actually not doing this correctly, the documentation confused me. When it says "safelist" is the only option, they mean that the "safelist" array *is* the option, not an object containing "safelist".

So:
```ts
htmlPurge([ 'carousel-item-next', 'carousel-item-next', 'carousel-item-prev' ]),
```
... would be correct, iirc.

It doesn't seem like the creator is around, so I've forked and reuploaded the module [here](https://github.com/Mexican-Man/vite-plugin-html-purgecss). This one uses the syntax as I presented in my original question, along with all other PurgeCSS options (which was what my underlying need was).
